### PR TITLE
Change group validation with a local variable

### PR DESCRIPTION
**Pull request by pubudu538**

### CODE REVIEW

File: components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java [hunk 1]
```diff
@@ -1622,6 +1626,8 @@ private static void validateGroupsRecursively(ConcurrentHashMap<String, Cartridg
                                                   Collection<CartridgeGroupReferenceBean> groups, boolean hasDP)
             throws RestAPIException {
 
+        boolean groupHasDP = false;
```
Comment (lahirus) [inline components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java]: Shall we s/hasDP/hasDeploymentPolicy?

Comment (pubudu538) [inline components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java]: Now I have changed those variables.

**Comment by imesh** [comment]
Hi Pubudu, the merge process failed with the following error, can you please try to update your local branch with the remote and update this pull request?

Auto-merging components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41.java
CONFLICT (content): Merge conflict in components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41.java
Failed to merge in the changes.
Patch failed at 0005 Fix Exception handling in addCartridgeGroup method

Thanks

**Comment by pubudu538** [comment]
Hi Imesh, I updated my local branch and merged with the master. Can you check now?

**Comment by imesh** [comment]
Thanks for the quick response! It looks like still we have a problem:

CONFLICT (content): Merge conflict in components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java
Failed to merge in the changes.
Patch failed at 0001 Adding a local variable for recursive group validate method

**Comment by pubudu538** [comment]
Hi Imesh,
I reverted all the commits made to the branch and committed again. I think now you should be able to merge. Sorry for the inconvenience caused.
Thank you.

**Comment by imesh** [comment]
Thanks Pubudu!! That worked, I have now merged this pull request to master branch.

**Pull request closed without merging**